csharp/ql/consistency-queries/DataFlowConsistency.ql

```diff
@@ -0,0 +1,54 @@
+import csharp
+import cil
+import semmle.code.csharp.dataflow.internal.DataFlowPrivate
+import semmle.code.csharp.dataflow.internal.DataFlowPublic
+import semmle.code.csharp.dataflow.internal.DataFlowImplConsistency::Consistency
+
+private class MyConsistencyConfiguration extends ConsistencyConfiguration {
```

Comment:
Why do we need all these exclusions? It would be great to see what other languages need to exclude.

Reply:
The exclusions in `uniqueEnclosingCallableExclude` and `missingLocationExclude` should eventually be fixed. The others are needed.